src/components/common/ColonyHome/ColonyMembers/MembersSubsection.tsx

```diff
@@ -85,14 +85,14 @@ const MembersSubsection = ({
 }: // maxAvatars = MAX_AVATARS,
 Props) => {
   const colonyWatchers = useMemo(() => watchers?.items || [], [watchers]);
-  const { user, wallet } = useAppContext();
```

Comment:
I notice you are removing the wallet retrieved from the useAppContext but it looks like we need the wallet address when using getAllUserRoles. Is there another way to get the wallet address or maybe getAllUserRoles will be removed?

Reply:
> I notice you are removing the wallet retrieved from the useAppContext

I am not removing it. I am in fact adding it. `useAppContext` does not exist in the Dapp, it's something new, only available in the CDapp, and it's the new way to get the user's `wallet` and `user` object.

> getAllUserRoles

This is not relevant anymore in the CDapp, and will be removed / refactored by #41

Reply:
PS: 

In this particular instance I may have removed the hook, since I didn't think it was required for this component _(Again, I did a rough pass over the components, just to get them render something)_

Feel free to put it back in, if you need values from it.